x-pack/packages/ai-infra/inference-common/src/errors.ts

```diff
@@ -51,9 +51,7 @@ export type InferenceTaskRequestError = InferenceTaskError<
 >;
 
 export function createInferenceInternalError(
-  message: string = i18n.translate('xpack.inference.internalError', {
-    defaultMessage: 'An internal error occurred',
-  }),
+  message = 'An internal error occurred',
```

Comment:
question: why did we un-internationalize this string?

Reply:
We never i18n-ize technical errors. ES doesn't, Kibana doesn't.